lib/players/human.rb

```diff
@@ -0,0 +1,7 @@
+class Human < Player
+
+  def move(token, board=nil)
+    input = gets.strip
+    return input
```

Comment:
I'm more used to it now, but this was my early 'ruby jitters'.  I'm more bent towards the python "explicit is better than implicit" way of specifying things.